src/firefly/java/edu/caltech/ipac/astro/FITSTableReader.java

```diff
@@ -615,10 +604,9 @@ private static String getAttributeValue(Object obj,
      * @param catName data group title (catName is not used).
      * @deprecated
      */
-    public static List<DataGroup> convertFITSToDataGroup(String FITS_filename, String catName)
+    public static List<DataGroup> convertFITSToDataGroup(String FITS_filename, String catName, int idx)
```

Comment:
Similar to `getStarTableList`, this should be done via an overloaded method.  i.e.
`public static DataGroup convertFITSToDataGroup(String FITS_filename, String catName, int idx)` 

If the original method is not used and cannot be easily implemented, I would just throw an unsupported exception for now.